packages/office-ui-fabric-react/package.json

```diff
@@ -23,6 +23,7 @@
     "code-style": "node ../../scripts/just.js code-style",
     "start": "cd ../../apps/fabric-website-resources && npm start",
     "start-test": "node ../../scripts/just.js jest-watch",
+    "load-theme": "node ./scripts/loadTheme.js",
```

Comment:
This is probably fine for a part 1 kind of some checkins. I'm looking for something that will gate it if it gets out of sync. Also, probably should rename this to something like "generateDefaultThemeSassFiles.js" or something